components/feed/FeedItem.tsx

```diff
@@ -23,6 +23,7 @@ const FeedItem = ({
   showLink: boolean;
 }) => (
   <div id={data.id}>
+    <div className="divider" />
```

Comment:
Can you somehow make the first line go away?
<img width="854" alt="Screenshot 2023-04-12 at 14 26 28" src="https://user-images.githubusercontent.com/7117993/231550817-a2dd0e9b-0ce3-42d5-b74b-71f684f13a90.png">
aka so there is no line between the comment box and the first comment?

Reply:
Maybe this??? https://tailwindcss.com/docs/divide-width#add-borders-between-horizontal-children